pydantic/__init__.py

```diff
@@ -16,6 +17,9 @@
 # WARNING __all__ from .errors is not included here, it will be removed as an export here in v2
 # please use "from pydantic.errors import ..." instead
 __all__ = [
+    # annotated types utils
+    'create_model_from_namedtuple',
```

Comment:
@samuelcolvin I didn't do it directly in this PR but I think it's quite nice to have the helpers exposed properly.
If you're ok with this I reckon we'll be able to add a `create_model_from_dataclass` instead of the not-very-pretty `pydantic.dataclasses.dataclass(StdDataclass).__pydantic_model__`

Reply:
yes agree, perhaps we might want `create_model_from` which takes any of the three types, but we definitely want these standalone methods for performance.

Or we could go even further and extend `create_model` to take a single dataclass, named tuple or typed dict instead of fields?